SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/BaseData.h

```diff
@@ -239,23 +239,23 @@ class SOFA_CORE_API BaseData : public DDGNode
     /// True if the value has been modified
     /// If this data is linked, the value of this data will be considered as modified
     /// (even if the parent's value has not been modified)s
-    SOFA_DEPRECATE_ASPECT("ExecParams are not needed anymore. Update your code by using isSet().")
+    SOFA_ATTRIBUTE_DEPRECATED_EXECPARAMS("Update your code by using isSet().")
```

Comment:
This is part of the DEPRECATING ASPECT movement. So not sure loosing that is a good idea. In addition SOFA_ATTRIBUTE_DEPRECATED_EXECPARAMS ... is misleading as execparams are by no way deprecated (and won't be)